packages/liquidator/src/liquidationStrategy.js

```diff
@@ -156,27 +168,68 @@ module.exports = (
       // 3) If WDF is active and withdraw has passed WDF activation percentage, then submit min liquidation
       emit("info", {
         message: "Liquidator bot is extending withdraw deadline, funds may need to be added to liquidate full position",
-        position: position,
+        position,
         maxLiquidationPrice: maxCollateralPerToken.toString(),
         syntheticTokenBalance: syntheticTokenBalance.toString(),
         maxTokensToLiquidateWei: maxTokensToLiquidateWei ? maxTokensToLiquidateWei.toString() : null,
         currentBlockTime,
         ...logInfo
       });
       tokensToLiquidate = minSponsorSize;
-    } else {
-      // 4) Do nothing.
     }
 
-    if (toBN(tokensToLiquidate).gt(toBN(0))) {
+    // If tokensToLiquidate is 0 at this point, determine why and send a customized log,
+    // and return undefined.
+    if (toBN(tokensToLiquidate).lte(toBN(0))) {
+      // Case 1: WDF is active but liveness threshold has not passed yet
+      if (
+        wdfActive &&
+        hasWithdrawRequestPending({ position, currentBlockTime }) &&
+        !passedDefenseActivationPercent({ position, currentBlockTime })
+      ) {
+        // If requested withdrawal was within the block window, then upgrade the log level.
+        // This makes it possible to emit only one log about skipping a liquidation
+        // due to the WDF activation threshold not being passed.
+        let logLevel = shouldEmitWDFSkipLogs ? "info" : "debug";
```

Comment:
does info trigger PD and debug does not?

Reply:
`info` triggers a Slack alert and debug does not. For sake of simplicity and little added benefit since no PD lett's just revert to debug

Reply:
for the record i am fine with either debug or info, just was concerned if it was a PD level alert

Reply:
If its info it will keep spamming the Slack channel so better to keep debug for now